webcompat/templates/list-issue/issue-list.html

```diff
@@ -1,34 +1,34 @@
-<div class="wc-content js-issue-list">
+<div class="wc-UIContent js-list-issue">
```

Comment:
Are we renaming all instances of `IssueList` or `issue-list` to `ListIssue` and `list-issue`? Or is there a difference between the two now?